src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -125,7 +125,8 @@
             "problemStatement": {
                 "title": "Problem Statement",
                 "taskCommand": "Task",
-                "testCaseCommand": "Insert Test Case"
+                "testCaseCommand": "Insert Test Case",
+                "tasksOfExercise": "Tasks of exercise:"
```

Comment:
```suggestion
                "tasksOfExercise": "Exercise tasks:"
```

Reply:
Maybe even just "Tasks"? There is only one exercise open.